sources/packages/backend/apps/api/src/route-controllers/application/application.students.controller.ts

```diff
@@ -97,11 +96,11 @@ export class ApplicationStudentsController extends BaseController {
   })
   async getByApplicationId(
```

Comment:
Not a part of the PR and can be absolutely ignored. Adding comments to the old methods when doing changes could be helpful.

Reply:
This one must be taken care of as part of the API reorganization in progress right now. The DTOs also need to be renamed and all.
@andrepestana-aot - FYI.